13_sandboxes/jupyter_sandbox.py

```diff
@@ -0,0 +1,69 @@
+# ---
+# cmd: ["python", "13_sandboxes/jupyter_sandbox.py"]
+# tags: ["use-case-sandboxed-code-execution"]
+# ---
+
+# # Run a Jupyter notebook in a Modal Sandbox
+#
+# This example demonstrates how to run a Jupyter notebook in a Modal [Sandbox](/docs/guide/sandbox).
+
+import secrets
+
+import modal
+
+# All Sandboxes are associated with an App. We look up our app by name, creating it if it doesn't exist.
+
+app = modal.App.lookup("example-jupyter", create_if_missing=True)
+
+# We define a custom Docker image that has Jupyter and some other dependencies installed.
+# Using a custom image allows us to avoid re-installing these deps on every Sandbox startup.
+
+image = (
+    modal.Image.debian_slim()
+    .pip_install("jupyter~=1.1.0", "bing-image-downloader~=1.1.2")
+    .pip_install("pandas~=2.2.3")  # Any other deps
```

Comment:
I like the callout to "other deps", but I don't think we need a separate pip install.

Maybe a commented-out `.pip_install(pandas, numpy, seaborn)`?

Reply:
ya good call